src/components/v5/frame/NavigationSidebar/partials/NavigationSidebarMobileContentWrapper/NavigationSidebarMobileContentWrapper.tsx

```diff
@@ -34,7 +34,7 @@ const NavigationSidebarMobileContentWrapper: FC<
       },
     )}
   >
-    <div className="inner pt-4 pb-6 h-full flex flex-col gap-6">
+    <div className="inner pt-4 pb-6 !px-4 h-full flex flex-col gap-6">
```

Comment:
Why is `px-4` with `!important` flag?

Reply:
Because I'm using `inner` class and I want to overwrite the padding that is added there